changelogs/fragments/4933-fix-rax-clb-nodes.yaml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - community.general.rax_clb_nodes - fix syntax to be compatible with Python3 (https://github.com/ansible-collections/community.general/pull/4933)
```

```suggestion
  - rax_clb_nodes - fix code to be compatible with Python 3 (https://github.com/ansible-collections/community.general/pull/4933).
```
(This has nothing to do with syntax, since there was no syntax error. The syntax was both valid in Python 2 and Python 3, what changed was how exactly the code works.)